leo_description/hooks/leo_description.dsv.in

```diff
@@ -0,0 +1 @@
+prepend-non-duplicate;IGN_GAZEBO_RESOURCE_PATH;share;@CMAKE_INSTALL_PREFIX@/share
```

Comment:
The second path is not needed as the first one is already relative to the package prefix path. Removing the second path also no longer requires the hook to be cmake-templated (we can remove the `.in` extension)
```suggestion
prepend-non-duplicate;IGN_GAZEBO_RESOURCE_PATH;share
```